mingw-w64-pico-sdk/PKGBUILD

```
@@ -0,0 +1,39 @@
+# Contributer: thewh1teagle
+_realname=pico-sdk
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.5.1
+pkgrel=1
+pkgdesc="Libraries for C/C++ development on RP2040 microcontrollers."
+arch=("any")
+mingw_arch=("mingw64" "ucrt64" "clang64")
+url="https://github.com/raspberrypi/pico-sdk"
+license=("BSD-3-Clause")
+makedepends=("git")
+options=("!strip")
+install=${_realname}-${MSYSTEM}.install
```

```suggestion
install=${_realname}.install
```
One file should be sufficient. Keep just one and rename it.